src/Helpers/Helper.php

```diff
@@ -63,6 +63,8 @@ public static function BuildSlackNotificationAttachment($op) {
 
         $fields[trans('calendar::seat.fleet_commander')] = $op->fc ? $op->fc : trans('calendar::seat.unknown');
         $fields[trans('calendar::seat.staging_system')] = $op->staging_sys ? $op->staging_sys : trans('calendar::seat.unknown');
+        $fields[trans('calendar::seat.staging_info')] = $op->staging_info ? $op->staging_info : trans('calendar::seat.unknown');
+		$fields[trans('calendar::seat.description')] = $op->description ? $op->description : trans('calendar::seat.unknown');
```

Comment:
This line need indentation to be fixed.
Also, maybe when no description is provided, it will be better to not disclosed the field ?

Reply:
Done